### PR TITLE
Feature/graph zoom number line

### PR DESCRIPTION
**Pull request by ssipasseuth**

### CODE REVIEW

File: views/js/pciCreator/dev/graphZoomNumberLineInteraction/runtime/css/graphZoomNumberLineInteraction.css [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+.graphZoomNumberLineInteraction .shape-controls .intervals-selected .interval .btn {margin-bottom:10px;}
+
+.graphZoomNumberLineInteraction .shape-controls button {padding: 0 6px 8px; height:36px; width:72px; margin-right:2px;}
+.graphZoomNumberLineInteraction .shape-controls button img {height:20px;width:50px;}
+
+.graphZoomNumberLineInteraction .shape-controls .intervals-template {display:none;}
+
+.graphZoomNumberLineInteraction .shape-controls .intervals-row {margin-bottom: 2px;}
+
+/*alternative deleter*/
```
Comment (clawfire) [inline views/js/pciCreator/dev/graphZoomNumberLineInteraction/runtime/css/graphZoomNumberLineInteraction.css]: If it's commented, delete it

File: views/js/pciCreator/dev/graphZoomNumberLineInteraction/runtime/graphZoomNumberLineInteraction.js [hunk 2]
```diff
@@ -0,0 +1,342 @@
...
+        //@todo make it responsive
+
+        return paper;
+    }
+
+    function buildAxisConfig(rawConfig){
+
+        var _color = rawConfig.graphColor || '#266d9c';
+
+        return {
```
Comment (clawfire) [inline views/js/pciCreator/dev/graphZoomNumberLineInteraction/runtime/graphZoomNumberLineInteraction.js:30]: so basicly just color and thickness can be configured ?

**Comment by clawfire** [comment]
Clean and well writen code as usual. Keep out, the new factory you wrote may need more documentation for a better understanding.

File: views/js/pciCreator/dev/graphZoomNumberLineInteraction/runtime/graphZoomNumberLineInteraction.js (hunk 2 above)
Comment (ssipasseuth) [inline views/js/pciCreator/dev/graphZoomNumberLineInteraction/runtime/graphZoomNumberLineInteraction.js:30]: Well those are not even part of the acceptance criteria. This is just to offer a way for customization in the future if need be.

**Comment by ssipasseuth** [comment]
Yes, although the code is working ok as it is, it may miss some functionalities. After the merge I will work on the details ;)

File: views/js/pciCreator/dev/graphZoomNumberLineInteraction/runtime/graphZoomNumberLineInteraction.js (hunk 2 above)
Comment (clawfire) [inline views/js/pciCreator/dev/graphZoomNumberLineInteraction/runtime/graphZoomNumberLineInteraction.js:30]: ok that's what we discuss, it's fine for me so.

**Pull request merged**